emmet-core/emmet/core/qc_tasks.py

```diff
@@ -0,0 +1,609 @@
+        None,
+        description="Parameters from a previous qchem calculation in the series",
+    )
+
+    lev_theory: LevelOfTheory = Field(
+        None, description="Level of theory used in the qchem calculation"
+    )
+
+    task_type: TaskType = Field(
+        None,
+        description="The type of the QChem calculation : optimization, single point ... etc.",
+    )
+
+    tags: Union[List[str], None] = Field(
+        [], title="tag", description="Metadata tagged to a given task."
+    )
+
+    solvent: Dict = Field(
+        None,
+        description="Dictionary with the implicit solvent model and respective parameters",
+    )
+
+    lot_solvent: str = Field(
+        None,
+        description="A string representation with the combined information of the solvemt used and the level of theory",
+    )
+
+    custom_smd: str = Field(
+        None, description="Parameter string for SMD implicit solvent model"
+    )
+
+    special_run_type: str = Field(
+        None, description="Special workflow name (if applicable)"
+    )
+
+    smiles: str = Field(
+        None,
+        description="Simplified molecular-input line-entry system (SMILES) string for the molecule involved "
+        "in this calculation.",
+    )
+
+    calc_type: CalcType = Field(
+        None,
+        description="A combined dictionary representation of the task type along with the level of theory used",
+    )
+
+    @classmethod
+    def from_qchem_calc_doc(cls, calc_doc: Calculation) -> "InputDoc":
+        """
+        Create qchem calculation input summary from a qchem calculation document.
+
+        Parameters
+        ----------
+        calc_doc
+            A QChem calculation document.
+
+        Returns
+        --------
+        InputDoc
+            A summary of the input molecule and corresponding calculation parameters
+        """
+        # TODO : modify this to get the different variables from the task doc.
+        return cls(
+            initial_molecule=calc_doc.input.initial_molecule,
+            rem=calc_doc.input.rem,
+            # lev_theory = level_of_theory(calc_doc.input.rem),
+            task_type=calc_doc.task_type,
+            tags=calc_doc.input.tags,
+            solvent=calc_doc.input.solvent,
+            # lot_solvent_string = calc_doc.input.lot_solvent_string,
+            # custom_smd = calc_doc.input.custom_smd,
+            # special_run_type = calc_doc.input.special_run_type,
+            # smiles = calc_doc.input.smiles,
+            calc_spec=calc_doc.calc_type,
+        )
+
+
+class CustodianDoc(BaseModel):
+    corrections: List[Any] = Field(
+        None,
+        title="Custodian Corrections",
+        description="List of custodian correction data for calculation.",
+    )
+
+    job: dict = Field(
+        None,
+        title="Custodian Job Data",
+        description="Job data logged by custodian.",
+    )
+
+
+# AnalysisDoc? Is there a scope for AnalysisDoc in QChem?
```

Comment:
I'm not sure. Is there a VASP analysis doc? If so, what's its purpose?

Reply:
Indeed [there is](https://github.com/materialsproject/emmet/blob/8df1e350d30aed8bdbbc9205375a79c331d97c32/emmet-core/emmet/core/tasks.py#L220). It is to report on things such as the change in volume over the course of the relaxation and the max force. The idea is that they are parameters that users want to query by to determine if the calculations seem sketchy, e.g. the volume changed too much or the force is too high.